### PR TITLE
Add DataFrame.applymap

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR adds DataFrame.`applymap`.

We can reuse `Series.apply`:

```python
>>> import pandas as pd
>>> pd.DataFrame({'a': [1,2,3], 'b': [1.0, 2.0, 3.0]})
   a    b
0  1  1.0
1  2  2.0
2  3  3.0
>>> pdf = pd.DataFrame({'a': [1,2,3], 'b': [1.0, 2.0, 3.0]})
>>> pdf.applymap(lambda x: x + 1)
   a    b
0  2  2.0
1  3  3.0
2  4  4.0
```

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -197,6 +197,59 @@ def _reduce_for_stat_function(self, sfun):
...
+
+        This method applies a function that accepts and returns a scalar
+        to every element of a DataFrame.
+
+        .. note:: unlike pandas, it is required for `func` to specify return type hint.
+
+        Parameters
+        ----------
+        func : callable
+            Python function, returns a single value from a single value.
```
Comment (thunterdb) [inline databricks/koalas/frame.py]: It is probably worth mentioning that this function must have python 3's style type hints to work, and to refer to the example for what that means (people are not necessarily aware of python 3's typing system).

Comment (thunterdb) [reply databricks/koalas/frame.py]: (and that only the return type hint is necessary, not the inputs)

Comment (ueshin) [reply databricks/koalas/frame.py]: There is already a note in the description above, but do we need it here too, or move it to here?
+1 for adding to refer to the example.

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -199,6 +199,63 @@ def _reduce_for_stat_function(self, sfun):
...
+        Apply a function to a Dataframe elementwise.
+
+        This method applies a function that accepts and returns a scalar
+        to every element of a DataFrame.
+
+        .. note:: unlike pandas, it is required for `func` to specify return type hint.
+            See https://docs.python.org/3/library/typing.html. For instance, as below:
+
+            >>> def function() -> int:
+            ...    1
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: `return 1`?

Comment (HyukjinKwon) [reply databricks/koalas/frame.py]: argh

**Comment by ChethanUK** [comment]
@HyukjinKwon is there any way to increase the number of threads in each Stage?

[https://i.imgur.com/O2BMkVA.png](Number of threads is 1)
Now, Only 1 thread per each stage.

Also, Why is max rec size is 100KB? 
Stage 0 contains a task of very large size (46976 KB). The maximum recommended task size is 100 KB.

**Pull request merged**